.eslintignore

```diff
@@ -8,3 +8,5 @@ pr-inactivity-bookmarklet.js
 question-bookmarklet.js
 flow/
 danger/
+bots/
+scripts/
```

Comment:
I'm worried about excluding them because most likely we'll never actually fix them and re-enable them. Could we instead either fix the errors or modify the eslint config to disables the linters we don't care about for those folders?

Reply:
Done. `eslint` only ignores the top `node_modules` directory by default, so using `bots/node_modules` here is sufficient to suppress the errors originating from third party code.